src/translations.php

```diff
@@ -138,6 +138,14 @@
         "Longest Week Streak" => "pi long tras semèn",
         "Present" => "Prezan",
     ],
+    "hy" => [
+        "Total Contributions" => "Ընդհանուր ներդրումը",
+        "Current Streak" => "Ընթացիկ շարք",
+        "Longest Streak" => "Ամենաերկար շարք",
+        "Week Streak" => "Ընթացիկ շաբաթների շարք",
+        "Longest Week Streak" => "Ամենաերկար շաբաթների շարք",
```

Comment:
It seems like the characters in Armenian are wider than average and don't wrap very well at the same number of characters, possibly line breaks (`\n`) could be used here to prevent the text from going too close to the boundaries or crossing them?

```suggestion
        "Total Contributions" => "Ընդհանուր\nներդրումը",
        "Current Streak" => "Ընթացիկ շարք",
        "Longest Streak" => "Ամենաերկար շարք",
        "Week Streak" => "Ընթացիկ\nշաբաթների շարք",
        "Longest Week Streak" => "Ամենաերկար\nշաբաթների շարք",
```

Reply:
Yeah, adding newlines is a great idea!